### ISSUE TITLE
whether the transforms in torchvision 0.3.0 can only handle cpu tensor

### ISSUE THREAD

**Issue by JZhaoCH** [pytorch/vision#1055]
* I test this code in python 3.5.4 , torch 1.0.0 and torchvision 0.2.1, it work well. Whether the code will crash when it normalizes gpu tensor in torch 1.1.0 and torchvision 0.3.0 (python 3.6)

* I mean the transforms in torchvision 0.2.1 can handle cpu tensor and gpu tensor, but the transforms in torchvision 0.3.0 can only handle cpu tensor, or is the problem of torchvision 0.3.0 in python 3.6

* the problem is found in [issue 2](https://github.com/JZhaoCH/cnnmrf-pytorch/issues/2) and [issue 1](https://github.com/JZhaoCH/cnnmrf-pytorch/issues/1)

* I didn't run this code in latest pytorch to test whether it will crash, because my cuda version is 8.0 and it's incompatible with latest pytorch
```
from torchvision import transforms
from PIL import Image
import torch

# read a image

img_path = '1.jpg'
img = Image.open(img_path).convert('RGB')
loader = transforms.Compose([transforms.ToTensor()])
img = loader(img)

# covert image to torch tensor and move it to gpu
gpu_device = torch.device('cuda')
img = img.to(gpu_device)


norm = transforms.Normalize(mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225))
content_image = norm(img)
```

**Comment by fmassa** [comment]
Hi,

Thanks for the report. This has been fixed in https://github.com/pytorch/vision/pull/787 and is currently present the latest version of torchvision (master), which needs to be compiled from source.

It will be available in the next release as well.
